src/WaterPokemon.java

```diff
@@ -0,0 +1,132 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class WaterPokemon extends Pokemon {
+    String name;
+    int level;
+    int hp;
+    String food;
+    String sound;
```

Comment:
Name, level, hp, food en sound zijn hier overbodig, omdat je ze al in de super hebt gedefinieerd. Ze worden krijgen heir ook nooit een waarde, omdat je het in de constructor enkel doorstuurt naaar de super constructor.